### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brianlheim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ An RFC is only a design document, and not a single line of code needs to be writ
 
 ## When should I file an RFC?
 
+If you have an idea for something and are unsure if it needs an RFC, the best way to find out is to just ask! Collaborators are strongly encouraged to talk about their ideas with other project members before submitting an RFC. This can also help gauge early support for the feature and work out any obvious issues or design decisions.
```
Comment (brianlheim) [inline README.md]: per email discussion

File: README.md [hunk 2]
```diff
@@ -26,19 +28,15 @@ RFCs should be filed for proposing "substantial" changes. The meaning of "substa
...
+- Anything unlikely to be controversial or in need of discussion
 
 ## Proposing an RFC
 
 ### Step 0: Decide if you should file an RFC
 
-The development community is small and has limited bandwidth. If there are a lot of RFCs open, it may be better to help advance existing discussions rather than pile on new ones.
-
-As an RFC author, it's best if you only have one RFC of yours open at any moment. This restriction is porous, but you are advised to avoid multitasking and be cognizant of how much the community's attention may be divided across other issues.
+As an RFC author, it's best if you only have one RFC of yours open at any moment. This restriction is not strictly enforced, but you are advised to be respectful of other contributors who would also like their RFCs to receive attention, and to be cognizant of the potential cost of spreading the community's attention thinly across a multitude of proposals.
```
Comment (brianlheim) [inline README.md]: the original wording felt a little too standoffish to me; i agree we don't want people to open an RFC for everything, but i also don't want to discourage people from doing it at all or make them feel guilty for doing so.

File: README.md [hunk 3]
```diff
@@ -99,19 +99,30 @@ Here are some guidelines:
 
 When the RFC seems ready to conclude, anyone may make a motion for a Final Comment Period. The Final Comment Period lasts 14 days, and has a *disposition* to either **pass** or **reject**.
 
-When should the Final Comment Period start? There doesn't need to be a consensus with everyone involved (which is usually impossible), but the RFC should be in a state where the author feels that its writing effort is complete, the arguments have been clearly articlated, discussions are resolved and drawbacks acknowledged in the RFC's text, and there isn't a strong consensus *against* the disposition.
+To make a motion for Final Comment Period, simply leave a comment on the pull request thread. The Final Comment Period should also be announced on the same forums as above: sc-users, sc-dev, and scsynth.org; and optionally Facebook and Slack's #dev channel. Here is a template:
```
Comment (brianlheim) [inline README.md]: per email discussion. i'm not strongly tied to the wording of the template, just tried to mimic the one above.

File: README.md [hunk 4]
```diff
@@ -99,19 +99,30 @@ Here are some guidelines:
...
 
 Implementation of an RFC can start before, during, or after it has been approved. In fact, if you propose an RFC, you don't need to be the person to implement it, but usually the implementer is the same person as the RFC author.
 
 ## About the RFC process
 
 This isn't a legal document, and you shouldn't spend a lot of time analyzing its wording. This process is intentionally quite informal and subjective. Situations will come up that may not have been anticipated by this article, and they should be handled on a case-by-case basis.
 
-If something is ambiguous in this article, that means it left up to the discussion. If something is a *persistent* cause of ambiguity and confusion, then it may be wise to amend the RFC process itself for clarification.
-
+If something is ambiguous in this article, that means it is left up to discussion. If something is a persistent cause of confusion, then it may be wise to amend the RFC process itself for clarification.
```
Comment (brianlheim) [inline README.md]: removed italics to make it feel a little less stuffy

**Comment by snappizz** [comment]
all changes look good, thanks!

**Pull request merged**